profiles/common/modules/features/multisite_business_indicators/multisite_business_indicators_community/tests/multisite_business_indicators_community.test

```diff
@@ -9,10 +9,12 @@
  */
 class BusinessIndicatorsCommunityQAWebTestCase extends DrupalWebTestCase {
   protected $profile = 'multisite_drupal_communities';
-  
-  // @ignore sniffer_namingconventions_validvariablename_lowercamelname
-  var $public_files_directory = NULL;
-  
+
+  protected $publicFilesDirectory = NULL;
```

Comment:
Unfortunately you cannot change the name of this property since it is defined in the parent class. It needs to be `$this->public_files_directory`. See `DrupalWebTestCase::prepareEnvironment()`.

To satisfy PHP CodeSniffer you can delete this line. It is not needed since it is already defined in the parent class. But this should be renamed back to `$public_files_directory` everywhere it is used or it won't work.